deploy/skaffold/Dockerfile

```diff
@@ -35,13 +40,28 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}
     tar -zxvf helm-${HELM_VERSION}-linux-amd64.tar.gz && \
     mv linux-amd64/helm /usr/local/bin/helm
 
-FROM gcr.io/distroless/base 
-WORKDIR /root/
+ENV CLOUD_SDK_VERSION 193.0.0
+RUN easy_install -U pip && \
+    pip install -U crcmod && \
```

Comment:
Do we use gsutil anywhere? We only need crcmod to make some gsutil operations faster.

Reply:
No, I can remove this. Wasn't sure if we were going to need upload anything once we were inside running the tests

Reply:
fixed